Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -1,12 +1,109 @@
 import SwiftUI
+import PayPalUI
 
 
 @available(iOS 13.0.0, *)
 struct SwiftUIPaymentButtonDemo: View {
 
+    @State private var pickerId = 0
+    @State private var buttonId = 0
+
+    @State private var fundingIndex = 0
+    private var fundingSources = PaymentButtonFundingSource.allCasesAsString()
+    @State private var selectedFunding = PaymentButtonFundingSource.allCases()[0]
+
+    @State private var colorsIndex = 0
+    @State private var colors = PayPalButton.Color.allCasesAsString()
+
+    @State private var edgesIndex = 0
+    private var edges = PaymentButtonEdges.allCasesAsString()
+    @State private var selectedEdge = PaymentButtonEdges.allCases()[0]
+
+    @State private var sizesIndex = 1
+    private var sizes = PaymentButtonSize.allCasesAsString()
+    @State private var selectedSize = PaymentButtonSize.allCases()[1]
+
+    @ViewBuilder
     var body: some View {
         ZStack {
-            Text("WIP - SwiftUI Button Customization View")
+            VStack {
+                Picker("Funding Source", selection: $fundingIndex) {
+                    ForEach(fundingSources.indices) { index in
+                        Text(fundingSources[index])
+                    }
+                }
+                .pickerStyle(SegmentedPickerStyle())
+                .onChange(of: fundingIndex) { _ in
+                    selectedFunding = PaymentButtonFundingSource.allCases()[fundingIndex]
+                    colors = getColorFunding(with: selectedFunding)
+                    colorsIndex = 0
+                    pickerId += 1 // Hack to change ID of picker. ID is updated to force refresh, https://developer.apple.com/forums/thread/127560
```

Comment:
Interesting find. I am guessing this didn't work:

> Wouldn't .id(myObject.selectedItem) achieve the same, without artifically chanding this pickerID ?

Also kind of a weird workaround, so I am okay with either, just curious

Reply:
@jaxdesmarais I tried it but it didnt work :(

Reply:
Bummer, but oh well - what a goofy bug so nice find on a solution!